src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.tsx

```diff
@@ -31,26 +32,37 @@ import {
 import { PhrasalTemplatePhraseProps } from './PhrasalTemplatePhrase.types';
 import { getFieldPlaceholders, getNewDefaultField } from '../PhrasalTemplate.utils';
 import { KEYWORDS } from '../PhrasalTemplateField/PhrasalTemplateField.const';
-
+import { PhrasalTemplateResponseValuePhraseSchema } from 'modules/Builder/pages/BuilderApplet/BuilderApplet.schema';
 export const PhrasalTemplatePhrase = ({
   name = '',
   responseOptions = [],
   index = 0,
   onRemovePhrase,
+  field,
 }: PhrasalTemplatePhraseProps) => {
   const { t } = useTranslation('app');
+  // Use `useEffect` to trigger validation when `fields` or form state changes
   const [removePopupOpen, setRemovePopupOpen] = useState(false);
   const [previewPopupOpen, setPreviewPopupOpen] = useState(false);
-  const [addItemMenuAnchorEl, setAddItemMenuAnchorEl] = useState<null | HTMLElement>(null);
+  const [previewPhraseDisabled, setPreviewPhraseDisabled] = useState(false);
 
+  const [addItemMenuAnchorEl, setAddItemMenuAnchorEl] = useState<null | HTMLElement>(null);
   const phraseFieldsName = `${name}.fields`;
   const { control, setValue, getValues, formState } = useCustomFormContext();
-
   const { fields, append, remove, swap } = useFieldArray({
     control: control as unknown as Control<{ values: TPhrasalTemplateField[] }>,
     name: phraseFieldsName as 'values',
   });
 
+  useEffect(() => {
+    const validateField = async () => {
+      const result = await validatePhraseField(field);
+      setPreviewPhraseDisabled(result);
```

Comment:
@farmerpaul this will need to be changed to 
```suggestion
      setPreviewPhraseDisabled(!result);
```
after updating `validatePhraseField`